merlin/main.py

```diff
@@ -626,6 +609,101 @@ def setup_argparse() -> None:
     )
     server_stop.set_defaults(func=process_server)
 
+    server_stop: ArgumentParser = server_commands.add_parser(
+        "restart",
+        help="Restart merlin server instance",
+        description="Restart server.",
+        formatter_class=ArgumentDefaultsHelpFormatter,
+    )
+    server_stop.set_defaults(func=process_server)
+
+    server_config: ArgumentParser = server_commands.add_parser(
+        "config",
+        help="Making configurations for to the merlin server instance.",
+        description="Config server.",
+        formatter_class=ArgumentDefaultsHelpFormatter,
+    )
+    server_config.add_argument(
+        "-ip",
+        "--ipaddress",
+        action="store",
+        type=str,
+        # default="127.0.0.1",
+        help="Set the binded IP address for the merlin server container.",
+    )
+    server_config.add_argument(
+        "-p",
+        "--port",
+        action="store",
+        type=int,
+        # default=6379,
+        help="Set the binded port for the merlin server container.",
+    )
+    server_config.add_argument(
+        "-pwd",
+        "--password",
+        action="store",
+        type=str,
+        # default="~/.merlin/redis.pass",
+        help="Set the password file to be used for merlin server container.",
+    )
+    server_config.add_argument(
+        "--add-user",
```

Comment:
is there a "remove-user" command? I can't find it

Reply:
There are add and remove user flags to the config command.